programs/psitest/main.cpp

```diff
@@ -559,9 +560,11 @@ struct callbacks
          result.filetype = wasi_filetype_regular_file;
          result.nlink    = stat.st_nlink;
          result.size     = stat.st_size;
+#ifndef __APPLE__
          result.atim     = to_nsec(stat.st_atim);
          result.mtim     = to_nsec(stat.st_mtim);
          result.ctim     = to_nsec(stat.st_ctim);
+#endif  // apple
```

Comment:
Does Apple support `st_atime` (second resolution instead of nanosecond resolution) or is the timespec available under another name (like `st_atimespec`)?

Reply:
It does, it has been updated for st_atimespec, etc